backend/inspirehep/hal/tasks.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+import structlog
+from celery import shared_task
+from sword2.exceptions import (
+    NotAcceptable,
+    NotFound,
+    PackagingFormatNotAvailable,
+    RequestTimeOut,
+)
+
+from inspirehep.hal.core.sword import create, update
+from inspirehep.hal.core.tei import convert_to_tei
+from inspirehep.records.api import LiteratureRecord
+
+LOGGER = structlog.getLogger()
+
+
+CELERY_HAL_PUSH_RETRY_ON_SWORD_EXCEPTIONS = (
+    RequestTimeOut,
+    NotFound,
+    PackagingFormatNotAvailable,
+    NotAcceptable,
```

Comment:
does it make sense to retry on all exceptions? If the result can't possibly change, it makes no sense retrying. `RequestTimeOut` probably yes, the others I'm not sure.